src/test/java/com/google/sps/ExperienceServletTest.java

```diff
@@ -94,31 +98,32 @@ public void tearDown() {
 
   @Test
   public void doGet_successfulPath_getsPlayerExp() throws IOException, LoggedOutException {
```

Comment:
The name of the test could be a bit more clear, maybe

doGet_PlayerEarnsMoreExperience_UponSuccessfulCompletionOfPath